target/linux/ath79/image/generic.mk

```diff
@@ -3307,6 +3307,19 @@ define Device/yuncore_xd3200
 endef
 TARGET_DEVICES += yuncore_xd3200
 
+define Device/yuncore_cpe830
+  SOC := qca9533
+  DEVICE_VENDOR := YunCore
+  DEVICE_MODEL := CPE830(D)
+  DEVICE_ALT0_VENDOR = KuWfi
+  DEVICE_ALT0_MODEL = CPE830(D)
+  IMAGE_SIZE := 16000k
+  IMAGES += tftp.bin
+  IMAGE/tftp.bin := $$(IMAGE/sysupgrade.bin) | yuncore-tftp-header-16m
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio kmod-ledtrig-gpio -swconfig
```

Comment:
`kmod-leds-gpio` is not needed. `CONFIG_LEDS_GPIO` is already compiled into the kernel.

Reply:
I already removed it from git, why is it still there ?